front/package.json

```diff
@@ -23,7 +23,7 @@
     "@amplitude/analytics-node": "^1.3.5",
     "@auth0/nextjs-auth0": "^3.5.0",
     "@dust-tt/client": "file:../sdks/js",
-    "@dust-tt/sparkle": "^0.2.265",
+    "@dust-tt/sparkle": "^0.2.266-rc-1",
```

Comment:
🤔

Reply:
Yes, didn't want to block sparkle, so once that one is ready I'll publish sparkle and update the version here 😄